src/database/migrations/20190729073412-create-genre.js

```diff
@@ -0,0 +1,23 @@
+const up = (queryInterface, Sequelize) => queryInterface.createTable('Genres', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: Sequelize.INTEGER
+  },
+  name: {
+    type: Sequelize.STRING
```

Comment:
Kindly add **allowNull: false** to genre **name** so that it would not accept null value